### DISCUSSION TITLE
Allow bash commands

### DISCUSSION THREAD

**Discussion by lcd1232** [dbeaver/dbeaver#11386] [Q&A] [answered]
Hello.

I want to do next:
- execute query
- wait 1 minute
- execute the same query
- do it until the query return 0 results

As I understand it cannot be made right now. That's why I think scripting or bash commands can help with it

**Comment by p91paul** [comment, marked as answer]
There is an arrow next to the refresh button that allows you to schedule refreshes every x seconds (it won't stop when you get 0 results though).
Also, most databases have their own scripting language (e.g. PL/SQL for Oracle) that already works in dbeaver
